package.json

```diff
@@ -14,7 +14,7 @@
     "url": "https://opencollective.com/lint-staged"
   },
   "engines": {
-    "node": "^14.13.1 || >=16.0.0"
+    "node": ">=16.0.0"
```

Comment:
This should be bumped to at least 16.14.0.

Reply:
Updated to `^16.14.0 || >=18.0.0` to match `execa`. Also added a script to the repo for easily listing dependencies' Node.js requirements, `scripts/list-dependency-node-versions.js`:

<img width="682" alt="Screenshot 2023-07-02 at 21 17 14" src="https://github.com/okonet/lint-staged/assets/11628889/e813fe9f-33b5-4270-94f9-45563ca14ff1">